lib/elixir/test/elixir/kernel/errors_test.exs

```diff
@@ -676,160 +733,169 @@ defmodule Kernel.ErrorsTest do
-        defmacro sample(num), do: num + :foo
-        def other, do: sample(1)
-      end
-      """)
+    assert [
+             {:erlang, :+, [1, :foo], _},
+             {Kernel.ErrorsTest.MacrosErrorStacktrace, :sample, 1, _} | _
+           ] =
+             rescue_stacktrace("""
+             defmodule Kernel.ErrorsTest.MacrosErrorStacktrace do
+               defmacro sample(num), do: num + :foo
+               def other, do: sample(1)
+             end
+             """)
   end
 
   test "macros function clause stacktrace" do
     assert [{__MODULE__, :sample, 1, _} | _] =
-      rescue_stacktrace("""
-      defmodule Kernel.ErrorsTest.MacrosFunctionClauseStacktrace do
-        import Kernel.ErrorsTest
-        sample(1)
-      end
-      """)
+             rescue_stacktrace("""
+             defmodule Kernel.ErrorsTest.MacrosFunctionClauseStacktrace do
+               import Kernel.ErrorsTest
+               sample(1)
+             end
+             """)
   end
 
   test "macros interpreted function clause stacktrace" do
     assert [{Kernel.ErrorsTest.MacrosInterpretedFunctionClauseStacktrace, :sample, 1, _} | _] =
-      rescue_stacktrace("""
-      defmodule Kernel.ErrorsTest.MacrosInterpretedFunctionClauseStacktrace do
-        defmacro sample(0), do: 0
-        def other, do: sample(1)
-      end
-      """)
+             rescue_stacktrace("""
+             defmodule Kernel.ErrorsTest.MacrosInterpretedFunctionClauseStacktrace do
+               defmacro sample(0), do: 0
+               def other, do: sample(1)
+             end
+             """)
   end
 
   test "macros compiled callback" do
-    assert [{Kernel.ErrorsTest, :__before_compile__, [%Macro.Env{module: Kernel.ErrorsTest.MacrosCompiledCallback}], _} | _] =
-      rescue_stacktrace("""
-      defmodule Kernel.ErrorsTest.MacrosCompiledCallback do
-        Module.put_attribute(__MODULE__, :before_compile, Kernel.ErrorsTest)
-      end
-      """)
+    assert [
+             {
+               Kernel.ErrorsTest,
+               :__before_compile__,
+               [%Macro.Env{module: Kernel.ErrorsTest.MacrosCompiledCallback}],
+               _
+             }
+             | _
+           ] =
+             rescue_stacktrace("""
+             defmodule Kernel.ErrorsTest.MacrosCompiledCallback do
+               Module.put_attribute(__MODULE__, :before_compile, Kernel.ErrorsTest)
+             end
+             """)
   end
 
   test "failed remote call stacktrace includes file/line info" do
     try do
       bad_remote_call(1)
     rescue
       ArgumentError ->
-        stack = System.stacktrace
-        assert [{:erlang, :apply, [1, :foo, []], []},
-                {__MODULE__, :bad_remote_call, 1, [file: _, line: _]} | _] = stack
+        stack = System.stacktrace()
+
+        assert [
+                 {:erlang, :apply, [1, :foo, []], []},
+                 {__MODULE__, :bad_remote_call, 1, [file: _, line: _]} | _
```

Comment:
Same here.

Reply:
Something like this? Also I'm bad at variable names so if you have better suggestions I'm all ears. :)
```
        assert [apply, bad_remote_call | _] = stack
        assert {:erlang, :apply, [1, :foo, []], []} = apply
        assert {__MODULE__, :bad_remote_call, 1, [file: _, line: _]} = bad_remote_call
```